### PR TITLE
Add login button to header, add back button/styling to edit page

### PR DESCRIPTION
**Pull request by Laur04**

### CODE REVIEW

File: tjhsst/templates/clubs/edit.html [hunk 1]
```diff
@@ -8,11 +8,12 @@ <h2>{{ club_form.errors }}</h2>
...
                 <div class="tbl-row">
                     <span class="tbl-cell bold" style="vertical-align:top;padding-right:5px;">{{ field.label_tag }}</span>
-                    <span class="tbl-cell">{{ field }}</span>
+                    <span class="tbl-cell" style="padding-top:5px">{{ field }}</span>
                 </div>
             {% endfor %}
         </div>
-        <input type="submit" value="Save">
+        <input style="background-color: #3366FF;padding:5px 10px;border:none;margin:3px;font-size:20px;" type="submit" value="Save">
+        <a href="{% url 'clubs:show' club.url %}"><input style="background-color:  rgba(221, 64, 58, 1);padding:5px 10px;border:none;margin:3px;font-size:20px;" type="button" value="Back"></a>
```
Comment (anonymoose2) [inline tjhsst/templates/clubs/edit.html]: 1. Please delete the `<input type="button">` and just make the back button a link.
2. If we do style buttons, then 1) we will want that to be site-wide so it's consistent and 2) we will probably want to copy Ion's styling, not make our own.

File: tjhsst/templates/base.html [hunk 2]
```diff
@@ -37,6 +37,10 @@ <h1>TJHSST</h1>
                             <button type="submit" class="search-button"><i class="fa fa-search"></i></button>
                         </form>
                     </li>
+                {% else %}
+                    <li>
+                        <span class="navbar-text" style="color:rgb(255,255,255)"><a href="{% url 'auth:login' %}">Login</a></span>
```
Comment (anonymoose2) [inline tjhsst/templates/base.html]: 1. I think you meant to add this to the `if` tag a few lines down.
2. From testing with this addition moved down there, this link feels out of place (and the default underline-on-hover behavior feels wrong). Perhaps further discussion is needed.
3. It turns out setting the color explicitly is unnecessary (it's set by Bootstrap). I know it's done in the username label a few lines down, but we should remove that too.

File: tjhsst/templates/clubs/edit.html [hunk 3]
```diff
@@ -8,11 +8,12 @@ <h2>{{ club_form.errors }}</h2>
             {% for field in club_form %}
                 <div class="tbl-row">
                     <span class="tbl-cell bold" style="vertical-align:top;padding-right:5px;">{{ field.label_tag }}</span>
-                    <span class="tbl-cell">{{ field }}</span>
+                    <span class="tbl-cell" style="padding-top:5px">{{ field }}</span>
```
Comment (anonymoose2) [inline tjhsst/templates/clubs/edit.html]: I don't feel like this is really necessary. Again, further discussion needed.

File: tjhsst/templates/clubs/edit.html (hunk 1 above)
Comment (Laur04) [reply tjhsst/templates/clubs/edit.html]: I fixed these issues and pushed, so this should be good.

**Pull request merged**